app/src/docs/v1.api-spec.yaml

```diff
@@ -83,10 +415,14 @@ components:
               example: Internal Server Error
             type:
               example: 'https://httpstatuses.com/500'
-    HelloDetail:
-      type: string
-      description: Hello content
-      example: Hello World!
+    ObjectDetails:
+      type: object
+      properties:
+        objId:
+          type: string
+          description: This Object instance uuid
+          format: uuid
+          example: 00000000-0000-0000-0000-000000000000
```

Comment:
Let's attempt to keep these object definitions in alphabetical order on the yaml manifest.